src/simplewallet/simplewallet.cpp

```diff
@@ -6423,7 +6420,20 @@ bool simple_wallet::show_transfers(const std::vector<std::string> &args_)
         std::string double_spend_note;
         if (i->second.m_double_spend_seen)
           double_spend_note = tr("[Double spend seen on the network: this transaction may or may not end up being mined] ");
-        message_writer() << (boost::format("%8.8s %6.6s %25.25s %20.20s %s %s %d %s %s%s") % "pool" % "in" % get_human_readable_timestamp(pd.m_timestamp) % print_money(pd.m_amount) % string_tools::pod_to_hex(pd.m_tx_hash) % payment_id % pd.m_subaddr_index.minor % "-" % note % double_spend_note).str();
+        
+        transfers.push_back({
+          tr("pool"),
```

Comment:
Not sure writing translated data to a formatted file is a good idea.

Reply:
Should be ok now.
I have used `tr()` only in `show_transfers` but not in `export_transfers`